README.md

```diff
@@ -31,8 +31,34 @@ This bundle is for multi-node deployments, for individual deployments for
 developers, use the smaller
 [kubernetes-core](http://jujucharms.com/kubernetes-core) bundle.
 
-## Deploy the bundle
+## Proxy configuration
+
+If you're operating behind a proxy, you'll need to configure your model
```

Comment:
How would the users know if they are behind a proxy? Can you write more words here about that? What does being "behind" a proxy mean? You mean the Juju controller or the charms themselves. Be a little more specific.